<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>884</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="startCam">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>340</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Webcam</string>
   </property>
  </widget>
  <widget class="QPushButton" name="captureCam">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>340</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Snapshot</string>
   </property>
  </widget>
  <widget class="QPushButton" name="selectImg1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Image 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="camera">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="fileBox1">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>531</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>File Path</string>
   </property>
  </widget>
  <widget class="QLabel" name="image1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="image2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>130</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Image One</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>110</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Image Two</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopCam">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>340</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QLabel" name="resultImage">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>130</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>110</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Result Image</string>
   </property>
  </widget>
  <widget class="QPushButton" name="selectImg2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Image 2</string>
   </property>
  </widget>
  <widget class="QLabel" name="fileBox2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>531</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>File Path</string>
   </property>
  </widget>
  <widget class="QPushButton" name="morphButton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>340</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Morph</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delauney">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Delauney</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
